Sources/NIONFS3/MountTypes+Mount.swift

```diff
@@ -0,0 +1,82 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the SwiftNIO open source project
+//
+// Copyright (c) 2021-2022 Apple Inc. and the SwiftNIO project authors
+// Licensed under Apache License v2.0
+//
+// See LICENSE.txt for license information
+// See CONTRIBUTORS.txt for the list of SwiftNIO project authors
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+//===----------------------------------------------------------------------===//
+
+import NIOCore
+
+// MARK: - Mount
+public struct MountCallMount: Equatable {
+    public init(dirPath: String) {
+        self.dirPath = dirPath
+    }
+
+    public var dirPath: String
+}
+
+public struct MountReplyMount: Equatable {
+    public init(result: NFS3Result<MountReplyMount.Okay, NFS3Nothing>) {
+        self.result = result
+    }
+
+    public struct Okay: Equatable {
+        public init(fileHandle: NFS3FileHandle, authFlavors: [RPCAuthFlavor] = [.unix]) {
+            self.fileHandle = fileHandle
+            self.authFlavors = authFlavors
+        }
+
+        public var fileHandle: NFS3FileHandle
+        public var authFlavors: [RPCAuthFlavor] = [.unix]
+    }
+
+    public var result: NFS3Result<Okay, NFS3Nothing>
+}
+
+extension ByteBuffer {
+    public mutating func readNFSCallMount() throws -> MountCallMount {
+        let dirPath = try self.readNFSString()
+        return MountCallMount(dirPath: dirPath)
+    }
+
+    public mutating func writeNFSCallMount(_ call: MountCallMount) {
+        self.writeNFSString(call.dirPath)
+    }
+
+    public mutating func writeNFSReplyMount(_ reply: MountReplyMount) {
+        self.writeNFSResultStatus(reply.result)
+
+        switch reply.result {
+        case .okay(let reply):
+            self.writeNFSFileHandle(reply.fileHandle)
+            precondition(reply.authFlavors == [.unix] || reply.authFlavors == [.noAuth],
+                         "Sorry, anything but [.unix] / [.system] / [.noAuth] unimplemented.")
```

Comment:
`.unix` == `.system`, that aliasing is also the same in the spec. I think it was "system" first (because UNIX was always "the system") but then Windows and friends added NFS support and I reckon the RFC process suggested to  make `.unix` the canonical one?